grid_map_pcl/include/grid_map_pcl/PclLoaderParameters.hpp

```diff
@@ -47,6 +47,11 @@ class PclLoaderParameters {
     double resolution_ = 0.1;
     unsigned int minCloudPointsPerCell_ = 2;
     unsigned int maxCloudPointsPerCell_ = 100000;
+    // 0: Smallest value among the average values ​​of each cluster
+    // 1: Mean value of the cluster with the most points
```

Comment:
What happend to type = 2?

Reply:
Oh i just copy ros2 version. Fixed in https://github.com/ANYbotics/grid_map/pull/329/commits/1f1ab6992491a1d09b4e3c9cf49e97fc81756092